exercises/say/tests/say.rs

```diff
@@ -13,13 +13,18 @@ fn test_zero() {
 // If the below test is uncommented, it should not compile.
 //
 /*
-#[test]
-fn test_negative() {
-    assert_eq!(say::encode(-1), String::from("won't compile"));
-}
+
+ #[test]
```

Comment:
this is changing more lines than I expect it to change. A space got added before each line. That makes it hard to understand what changed, since now each line of this test is marked as a changed line. If this change (add space to each line) needs to be made for some reason, it would want to be a separate commit